FrontEnd/src/components/basics/ImageUpload/ImageUpload.module.scss

```diff
@@ -0,0 +1,71 @@
+$border-radius: 0.1875rem;
+.image_input {
+  display: none;
+}
+
+.image_input_label {
+  cursor: pointer;
+}
+
+.delete_wrapper_background {
+  position: absolute;
+  display: none;
+  width: 100%;
+  height: 100%;
+  background: rgba(32, 32, 32, 0.6);
+}
+
+.delete_wrapper {
+  position: absolute;
+  height: 2.5rem;
+  top: 50%;
+  left: 50%;
+}
+
+.delete {
+  position: relative;
+  top: -50%;
+  left: -50%;
+}
+
+.single_image {
+  height: 11rem;
+  width: 11rem;
+  position: relative;
+  @include bp('xs') {
```

Comment:
Add new lines between @includes and classnames (i.e. new line above .round_image, etc). There's other spots in the file that need new lines too (very similarly) but I won't comment on them to reduce repetitive comments

Reply:
agreed. I saw you put include at the end as well in other classes. Did the same